### DISCUSSION TITLE
bigfix/11.0/platform/Platform_pdfguides

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#80] [Q&A]
# bigfix/11.0/platform/Platform_pdfguides

Following is a list of links to the BigFix Platform user guides in PDF format:

https://help.hcl-software.com/bigfix/11.0/platform/Platform_pdfguides.html

**Comment by KunioMatsuo** [comment]
Following links to PDF are dead.
BigFix Installation Guide
BigFix Configuration Guide
BigFix Console Operator's Guide
